docs/getting-started.md

```diff
@@ -114,15 +114,15 @@ By tailoring these filters, you can obtain a variety of demographic insights fro
 Each theme within our API is bolstered by associated supporting tables. These tables are essential for understanding the range of possible values you can work with in the theme-specific tables. For example, if you're filtering by sector, such as `sector=Nutrition`, you'll want to know what sectors are available.
 
 
-You can retrieve a list of possible age ranges by querying the `sector` support table like so:
+You can retrieve a list of possible sectors by querying the `sector` support table like so:
 
 
 ```plaintext
 https://hapi.humdata.org/api/v1/metadata/sector?output_format=json&offset=0&limit=1000&app_identifier=Z2V0dGluZ3Mtc3RhcnRlZDpzaW1vbi5qb2huc29uQHVuLm9yZw==
 ```
 
 
-This functionality is not limited to age ranges. There are similar support tables for a variety of filters such as organizations, genders, sectors, and more. Querying these support tables provides you with the necessary information to apply precise filters and extract the data that's most relevant to your needs.
+This functionality is not limited to sectors. There are similar support tables for a variety of filters such as organizations, genders, sectors, and more. Querying these support tables provides you with the necessary information to apply precise filters and extract the data that's most relevant to your needs.
```

Comment:
@turnerm Should we remove "sectors" from the "others"  list (in the 2nd sentence), since it's now mentioned in the first sentence ?